DEVELOPER_GUIDE.md

```diff
@@ -65,13 +65,13 @@ OpenSearch builds using Java 11 at a minimum. This means you must have a JDK 11
 
 Download Java 11 from [here](https://adoptium.net/releases.html?variant=openjdk11).
 
-#### JDK 8 and 14
+#### JDK 14
 
-To run the full suite of tests, download and install [JDK 8](https://adoptium.net/releases.html?variant=openjdk8) and [JDK 14](https://jdk.java.net/archive/) and set `JAVA8_HOME`, `JAVA11_HOME`, and `JAVA14_HOME`. They are required by the [backwards compatibility test](./TESTING.md#testing-backwards-compatibility).
+To run the full suite of tests, download and install [JDK 14](https://jdk.java.net/archive/) and set `JAVA11_HOME`, and `JAVA14_HOME`. They are required by the [backwards compatibility test](./TESTING.md#testing-backwards-compatibility).
```

Comment:
@martin-gaievski this is interesting, we are not going to backport this change to 1.x, right? In this case, I am not sure we have to drop the JDK-8 requirements for bwc tests, did you have a chance to check it?

Reply:
@reta As to my knowledge this is not going to be back-ported to 1.x. It's meant to be for 2.0 only as per original requirements from [110](https://github.com/opensearch-project/opensearch-plugins/issues/110). 
I've run bwcTest locally for this branch with only JAVA11_HOME and JAVA14_HOME set and it passed without errors.

Reply:
All set, thank you